pysmt/smtlib/parser/__init__.py

```diff
@@ -80,7 +80,7 @@
     # no ambiguity when importing the parser: the only way to load the
     # cython version is by the so_path that targets .pyxbld .
     #
-    import imp
+    import implib
```

Comment:
This fix does not seem to work with cython: https://dev.azure.com/micheliandrea/PySMT/_build/results?buildId=548&view=logs&jobId=5b3fdf3a-600d-577c-6029-5dbd4ada42e3&j=5b3fdf3a-600d-577c-6029-5dbd4ada42e3&t=4c81876a-3912-5544-1861-6e026cdb91ca

Reply:
Done